### ISSUE TITLE
activity feed

### ISSUE THREAD

**Issue by JOHNKIMBLE** [elendil7/cycle-mania#6]
hey there sorry for the random issue.
Love this bot and its amazing documentation and notation. I am no programmer but fine with flat files like json and scripting pretty comfortably.  

Got this running on my unraid server pointing to local mongodb and working for our bike teams discord server.

I see in my dozzle that getting activity feed on the cronjob is catching a undefined exception likely at getRichClubActivities() from one of the interfaces.

![image](https://github.com/elendil7/cycle-mania/assets/9062306/52a7c6bf-894b-455c-8bea-67b0d32f7392)

I see in your roadmap you had Zwift (workout & activities platform) integration planned and don't mind trying at it in my free time, but never built out interfaces like this. 

Basically I wanted to see if you remembered what was missing from the interface still or how I could debug this error further. 
Never built out advanced interface and objects like this, but wanted to add what I could discover.
